woocommerce/woocommerce-android
# [Better Tablet Products] Removing of a picture is not saved

Closes: #11113


### Description

* Removed hiding of a "fake" toolbar with trash icon on the product image screen to avoid confusion that some users have
* Fixed a bug when if a user uses toolbar bar arrow navigation after image removing it was not saved

### Testing instructions

* Products -> Products with images
* Open an image -> remove it
* Go back using toolbar back arrow
* Notice that the change was reflected

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/ed97e6e4-523b-42ee-ba83-eb7d07a052ed
